test/max.test.js

```diff
@@ -0,0 +1,8 @@
+import test from 'ava'
+import {max} from '../src'
```

Comment:
Ah, the reason you're having this problem is because you haven't added the `import/export` to [`src/index.js`](https://github.com/eggheadio-github/stack-overflow-copy-paste/blob/master/src/index.js) yet. :)